difflint/lib.py

```diff
@@ -249,7 +197,25 @@ def detect_new_diff_lint_errors(log_output):
 
 
 def main():
-    if not lint_script_exists() or not all_files_configured():
+    parser = argparse.ArgumentParser(description='Linter that will examine ' +
+        'only new changes as you commit them.')
+    parser.add_argument('-c', '--check', action='store_true',
```

Comment:
That sounds reasonable to me. I don't have strong feelings about this, and I don't think it's worth blocking on. My bias in the long-term would be to remove `--check` and add appropriate dependencies to cover the configuration files or bundle them directly as part of this package.